js/src/components/adaptive-form/adaptive-form.js

```diff
@@ -130,7 +163,30 @@ function AdaptiveForm( { children, ...props }, ref ) {
 					setImmediate( () => setDelegation( batchQueue.shift() ) );
 				}
 
-				return children( formProps );
+				// Enhancements related.
+				formContext.handleSubmit = function ( event ) {
+					adapterRef.current.submitter = event.currentTarget;
+					return handleSubmit.call( this, event );
+				};
```

Comment:
Would be nice to explain this chunk a little bit more. Instead of `// Enhancements related.` that is a bit generic, I would add something more descriptive.

Reply:
Added in https://github.com/woocommerce/google-listings-and-ads/pull/1794/commits/7ccbc8f8e5568a6f74bbd15d337fbae286ca05f6.